File: app/src/main/java/com/cvv/fanstaticapps/randomticker/LicenseActivity.java
package com.cvv.fanstaticapps.randomticker;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.View;
import android.webkit.WebView;

import com.cvv.fanstaticapps.randomticker.activities.BaseActivity;

import butterknife.BindView;

public class LicenseActivity extends BaseActivity {
    @BindView(R.id.webview)
    WebView webView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_license);
    }

    @Override
    protected void onPostCreate(@Nullable Bundle savedInstanceState) {
        super.onPostCreate(savedInstanceState);
        webView.setLayerType(View.LAYER_TYPE_SOFTWARE, null);
        webView.loadUrl("file:///android_asset/licenses.html");
    }
}

File: app/src/main/java/com/cvv/fanstaticapps/randomticker/activities/LicenseActivity.kt
package com.cvv.fanstaticapps.randomticker.activities

import android.os.Bundle
import android.view.View
import com.cvv.fanstaticapps.randomticker.R
import kotlinx.android.synthetic.main.activity_license.*
import kotlinx.android.synthetic.main.app_bar.*

class LicenseActivity : BaseActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_license)

        setSupportActionBar(toolbar)
        supportActionBar?.setDisplayHomeAsUpEnabled(true)

        webView.setLayerType(View.LAYER_TYPE_SOFTWARE, null)
        webView.loadUrl("file:///android_asset/licenses.html")
    }
}
